### DISCUSSION TITLE
Getting an overwhelming amount of a single card type

### DISCUSSION THREAD

**Discussion by zeroeightysix** [mortii/anki-morphs#246] [Q&A] [answered]
I've been using anki-morphs to study Danish: for that, I've got a deck of [8000 sentences](https://ankiweb.net/shared/info/2018379287) with translations from danish to english and two card types (danish->english, english->danish).

Because there's a lot of sentences in the deck, I don't want to study all notes, but introduce ones with new words on them, so I've enabled the "Move new cards without unknown morphs to the end of the due queue" option.

Unfortunately, while studying, I'm only really getting the danish->english cards and (almost) never the english->danish ones, making my knowledge rather lopsided. The rest of my settings are on the defaults (aside from the word filters, of course).

What could be causing this? Is anki-morphs skipping the english->danish cards due to the morph in it already having been studied today (in the danish->english card), making it so that card never gets learned? How do I get a more balanced queue?

**Comment by aleksejrs** [comment]
I guess both cards of a note get the same queue position (AM's limitation), and then you see the first card of each note first. What happens to the other card except for being buried each day you review the first card  probably depends on AM's settings.

**Comment by mortii** [comment, marked as answer]
> What could be causing this? Is anki-morphs skipping the english->danish cards due to the morph in it already having been studied today (in the danish->english card), making it so that card never gets learned? How do I get a more balanced queue?

If you unselect the [skip options](https://mortii.github.io/anki-morphs/user_guide/setup/settings/skip.html), then that won't happen. Also check if you have any "bury siblings" options enabled in anki or other add-ons.
